src/main/java/edu/greenblitz/robotName/utils/shootingCalculations/ShooterAngle.java

```diff
@@ -0,0 +1,24 @@
+package edu.greenblitz.robotName.utils.shootingCalculations;
+
+
+import edu.greenblitz.robotName.FieldConstants;
+import edu.greenblitz.robotName.utils.GBMath;
+import edu.wpi.first.math.geometry.Rotation2d;
+import edu.wpi.first.math.geometry.Translation2d;
+import edu.wpi.first.math.geometry.Translation3d;
+
+public class ShooterAngle {
+
+    public static Rotation2d getShooterAngleBasedOnPosition(Translation3d position) {
+        Translation2d relativeSpeakerPosition = getRelativePositionOfMiddleOfSpeaker(position);
+        double angle = Math.atan(relativeSpeakerPosition.getY() / relativeSpeakerPosition.getX());
```

Comment:
consider changing angle to a more informative name, but this is not a must

Reply:
It is a must now